android/src/newarch/java/com/reactnativefastshadow/FastShadowViewManager.java

```diff
@@ -9,61 +9,50 @@
 import com.facebook.react.uimanager.annotations.ReactProp;
 import com.facebook.react.views.view.ReactViewGroup;
 import com.facebook.react.views.view.ReactViewManager;
+import com.facebook.react.module.annotations.ReactModule;
 
+@ReactModule(name = FastShadowViewManagerImpl.NAME)
```

Comment:
The only difference between this file and the one in `oldarch` is this annotation. Is there any way we could use the same file for both archs? Can we use `@ReactModule` with the non-Fabric arch?

Reply:
hi sorry for late response, I just following guide from here (https://github.com/react-native-community/RNNewArchitectureLibraries/tree/feat/back-fabric-comp), I'll look into it to make same file for both archs